osquery/core/tests/conversions_tests.cpp

```diff
@@ -55,7 +55,18 @@ TEST_F(ConversionsTests, test_utc) {
   FLAGS_utc = true;
   auto t2 = getUnixTime();
   auto rt2 = std::time(nullptr);
-  auto at2 = std::mktime(std::gmtime(&rt2));
+
+#ifdef WIN32
```

Comment:
This is blocked off using `#ifdef WIN32` gates because using `std::gmtime` on Windows triggers a insecure API warning. So, for this case, I don't think `isPlatform(TYPE_WINDOWS)` will work.